postgresql/_database.sls

```diff
@@ -39,6 +39,21 @@ postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}:
     - user: root
     {%- endif %}
 
+{%- for extension in database.get('extensions', []) %}
```

Comment:
I think it should be dict with possibility to specify enabled and possibly other options (eg. schema?)
There's already metadata structure used in postgresql.server that should be reused.